client/src/components/EventTypeDialog.js

```diff
@@ -102,17 +113,17 @@ export default function EventTypeDialog() {
             .then(res => {
                 status = res.status;
                 if (status < 500) return res.json();
-                else throw Error("Server error");
+                else setAlert(true, "Server error", "error")
             })
```

Comment:
I think we can keep these `throw Error` here and deal with all of them at the end in `.catch`

Reply:
Good point!